repokid/cli/repokid_cli.py

```diff
@@ -217,8 +224,8 @@ def _get_aardvark_data(aardvark_api_location, account_number=None, arn=None):
     while True:
         params = {'count': PAGE_SIZE, 'page': page_num}
         try:
-            r_aardvark = requests.post(aardvark_api_location, params=params, json=payload)
-        except requests.exceptions.RequestException as e:
+            r_aardvark = aardvark_client.post('/api/v1/advisor', params=params, data=payload)
```

Comment:
suggestion: should we create a new aardvark service module with the lyft specific code, so that it will be easier to upstream the changes? so here it would just be `r_aardvark = aardvark_service.post(...)` and then in the aardvark_service, we will place the lyft specific logic.

Reply:
Sounds good!